decidim-core/decidim-core.gemspec

```diff
@@ -21,49 +21,49 @@ Gem::Specification.new do |s|
   s.add_dependency "active_link_to", "~> 1.0"
   s.add_dependency "autoprefixer-rails", "~> 8.0"
   s.add_dependency "batch-loader", "~> 1.2"
-  s.add_dependency "carrierwave", "~> 1.1"
+  s.add_dependency "carrierwave", "~> 1.3"
   s.add_dependency "cells-erb", "~> 0.1.0"
   s.add_dependency "cells-rails", "~> 0.0.9"
-  s.add_dependency "charlock_holmes", ">= 0.7.6"
+  s.add_dependency "charlock_holmes", "~> 0.7"
   s.add_dependency "date_validator", "~> 0.9.0"
   s.add_dependency "devise", "~> 4.6"
   s.add_dependency "devise-i18n", "~> 1.2"
-  s.add_dependency "doorkeeper", "~> 4.4"
+  s.add_dependency "doorkeeper", "~> 5.1"
   s.add_dependency "doorkeeper-i18n", "~> 4.0"
   s.add_dependency "etherpad-lite", "~> 0.3"
   s.add_dependency "file_validators", "~> 2.1"
   s.add_dependency "foundation-rails", "~> 6.4", "< 6.5"
   s.add_dependency "foundation_rails_helper", "~> 3.0"
-  s.add_dependency "geocoder", "~> 1.4"
-  s.add_dependency "hashdiff", "~> 0.3.7"
-  s.add_dependency "invisible_captcha", "~> 0.10.0"
+  s.add_dependency "geocoder", "~> 1.5"
+  s.add_dependency "hashdiff", ">= 0.4.0", "< 2.0.0"
+  s.add_dependency "invisible_captcha", "~> 0.12"
   s.add_dependency "jquery-rails", "~> 4.3"
-  s.add_dependency "kaminari", "~> 1.0"
-  s.add_dependency "loofah", "~> 2.0", ">= 2.2.1" # version 2.2.0 has a vulnerability
-  s.add_dependency "mini_magick", "~> 4.8"
+  s.add_dependency "kaminari", "~> 1.1"
+  s.add_dependency "loofah", "~> 2.2"
+  s.add_dependency "mini_magick", "~> 4.9"
   s.add_dependency "mustache", "~> 1.1.0"
-  s.add_dependency "nobspw", "~> 0.4.0"
-  s.add_dependency "omniauth", "~> 1.6"
-  s.add_dependency "omniauth-facebook", "~> 4.0"
-  s.add_dependency "omniauth-google-oauth2", "~> 0.5.0"
+  s.add_dependency "nobspw", "~> 0.6.0"
+  s.add_dependency "omniauth", "~> 1.0"
```

Comment:
@josepjaume Shouldn't this be 1.9 instead of 1.0? This is important because that version includes a security fix: https://github.com/omniauth/omniauth/releases/tag/v1.9.0

Reply:
Yup, my bad, I'll fix it in another PR

Reply:
Fixed at #5233